### ISSUE TITLE
add plotting demos to python API

### ISSUE THREAD

**Issue by jbfaden** [autoplot/python#1]
Bob requested that the API presented to Python would include some plotting as well.  I've never done this in IDL and Matlab, but it should be easy to do.

**Comment by jbfaden** [comment]
I've made it so the pip autoplot package now includes the bridge to Autoplot, so you can say "applot([...],[...])".

**Comment by rweigel** [comment]
What was the commit id? Probably want to add a demo too.

I'd make the function name "plot" and use at as either

```
import autoplot as ap # Should be more common usage
ap.plot(...,...) # should call ap.javaaddpath() as needed
```

or

```
from autoplot import plot
plot(...,...) # should call ap.javaaddpath() as needed
```

**Comment by jbfaden** [comment]
The problem with automatically calling javaaddpath automatically is how is the jar path specified?  I thought of including a no-args version of javaaddpath which would use a default (http://autoplot.org/latest/autoplot.jar) version, or maybe an environment variable.

**Comment by jbfaden** [comment]
Originally I had an applot which would use the jpype version of Java, the one running in the session.  The problem with this is Macs have the strange issue where they can't bring up GUIs, but also I found that on Linux exiting Autoplot exits the entire python session.   So the new version of applot (I used the name from the IDL version) serializes the data and sends a plot command to a running external Autoplot session.
